routes/index.js

```diff
@@ -9,12 +9,36 @@ var express = require('express'),
 var fs = require('file-system');
 var officegen = require('officegen');
 var async = require('async');
+var nodemailer = require('nodemailer')
+var config = require('config')
+
+router.post('/contactus', function(req, res, next) {
+    var name = req.body.name
+    var email = req.body.email
+    var question = req.body.question
+    var smtpTransport = nodemailer.createTransport('SMTP', {
+        service: 'SendGrid',
+        auth: {
+            user: process.env.SENDGRID_USER || config.get('emailVerification.user'),
+            pass: process.env.SENDGRID_PASS || config.get('emailVerification.pass')
+        }
+    });
+    var mailOptions = {
+        to: 'jonathanardewanta@hotmail.com',
```

Comment:
should this go to a different address?. Also you may want to make the subject more informative.
"Question about glorify.cc" or something.